src/main/java/com/sap/sgs/phosphor/fosstars/data/github/UsesOwaspDependencyCheck.java

```diff
@@ -20,11 +18,15 @@
 
 /**
  * This data provider checks if an open-source project uses OWASP Dependency Check Maven plugin to
- * scan dependencies for known vulnerabilities. If it does, the provider sets {@link
- * com.sap.sgs.phosphor.fosstars.model.feature.oss.OssFeatures#SCANS_FOR_VULNERABLE_DEPENDENCIES}
- * to true.
+ * scan dependencies for known vulnerabilities.
  */
-public class UsesOwaspDependencyCheck extends AbstractGitHubDataProvider {
+public class UsesOwaspDependencyCheck extends CachedSingleFeatureGitHubDataProvider {
+
+  /**
+   * A feature which is filled out by the provider.
+   */
+  public static final Feature<Boolean> USES_OWASP_DEPENDENCY_CHECK
```

Comment:
Shouldn't this be in OssFeatures.java?

Reply:
Good question. At first, I was going to add this to `OssFeatures`. In fact, it make sense to have it there. On the other hand, it's not clear where else we can use this new feature. In the data providers, this feature is used to fill out `SCANS_FOR_VULNERABLE_DEPENDENCIES`. So, defining this new feature in the provider is enough. Let's have it here, and we can move it to `OssFeatures` if necessary.